infrastructure/monitoring/kibana/alert-rules.json

```diff
@@ -0,0 +1,79 @@
+[
```

Comment:
This file defines alert rules, i.e. what conditions constitute as alerts (>70% CPU in use, >50% disk space used).
The rule format follows the payload of Kibana's [Create rule API](https://www.elastic.co/guide/en/kibana/current/create-rule-api.html). The easiest way of creating a new rule is to do it through the Kibana UI and copy the payload it send to the API :) 

See `infrastructure/monitoring/kibana/setup-alerts.js` to understand how we read this file every time the stack is deployed.